templates/JKG-ec2-v5.yaml

```diff
@@ -0,0 +1,20 @@
+# Instance to test integration with parkmycloud
+template_path: managed-ec2-v5.yaml
+stack_name: JKG-SB-test-ec2
+parameters:
+  # The Sage deparment for this resource
+  Department: "sysbio"
+  # The Sage project this resource will be used for
+  Project: "amp-ad"
+  # The resource owner
+  OwnerEmail: "jake.gockley@sagebase.org"
+  # EC2 instance type (available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "t2.xlarge" <- choose a new type
```

Comment:
"<- choose a new type" is invalid.  if you intend that to be a comment then comments are prepended by '#' character